pop-api/integration-tests/contracts/fungibles/lib.rs

```diff
@@ -7,7 +7,10 @@
 ///
 use ink::prelude::vec::Vec;
 use pop_api::{
-	assets::fungibles::{self as api},
+	assets::fungibles::{
+		self as api,
+		events::{Approval, ClearMetadata, Create, SetMetadata, StartDestroy, Transfer},
```

Comment:
Generally accepted convention for event naming is past tense - i.e. Created, MetadataSet etc.

Assume this is based on the convention set by ERC20/PSP22

Reply:
Correct